Dockerfile

```diff
@@ -10,7 +10,7 @@
 #
 
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
-FROM registry.access.redhat.com/ubi8-minimal:8.4-200 as builder
+FROM registry.access.redhat.com/ubi8-minimal:8.4-200
```

Comment:
I'm not sure why running `olm/update_resources.sh` made this change?

Reply:
It is fixed in main branch. Pls rebase.

Reply:
Probably regexp doesn't work correctly ^(